### ISSUE TITLE
Add license file

### ISSUE THREAD

**Issue by felipedaragon** [da-baranov/SciDe#4]
Can you please add a license file for this project? :) I'd suggest the MIT license http://opensource.org/licenses/MIT

**Comment by da-baranov** [comment]
Ok, but my project is just a wrapper around original Sciter HTML engine owned by Andrew (http://sciter.com). I don't pretend to be a co-author and would like to redirect anyone who want to have a licence to the original licence publshed at http://sciter.com/sciter/sciter-source-price.whtm

**Comment by felipedaragon** [comment]
Yes. I'm aware of this. The license is just for covering the wrapper code usage.
